### PR TITLE
[DO NOT MERGE - INCOMPLETE] Added Onfido iOS SDK with integration to iOS React Native Sample App

### PR DESCRIPTION
**Pull request by anuragajwani**
# Problem 

There is no example on how to integrate Onfido iOS SDK into React Native

# Solution

Add example on how to integrate with Onfido iOS SDK using React Native

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -31,10 +31,7 @@ export default class App extends Component<{}> {
   }
 
   launchSDK() {
-    NativeModules.OnfidoSDK.startSDK(
-      (applicantId) => { this.setTextContent("Verification complete", "To perform another verification, press \"Launch\"") },
-      (errorCause) => { this.setTextContent("Flow not finished", "To try again, press \"Launch\"") }
-    );
+    NativeModules.OnfidoSDK.startSDK();
```
Comment (sergiomarqmoura) [inline App.js:34]: can we maybe have 2 different buttons to launch both SDKs? these callbacks are needed to update the UI after the flow completion

Comment (anuragajwani) [reply App.js:34]: @sergiomarquesmoura this PR is not to merge yet it is incomplete (on purpose) but here for visibility.

Comment (anuragajwani) [reply App.js:34]: added `[DO NOT MERGE - INCOMPLETE]` for calrity

**Comment by rfreitas** [comment]
how do you run it?

**Comment by anuragajwani** [comment]
@rfreitas go into the `ios` directory. Run `pod install` and then open the `ReactNativeSampleApp.xcworkspace` using `Xcode`. Select `ReactNativeSampleApp` and a physical ios device to run it on and then select `Product` from the menu bar and then `Run`. 

You can find the instructions in the `README.md`.

**Comment by theleanship** [comment]
My ios knowledge is lacking but I thought I'd try get this running before digging into it properly. Seem to be hitting an swift version incompatibility in dependencies (onFido andSwiftyJSON) , all I see in the Podfile is the onFido pod though so I wonder if you're aware of this already?

This is in Xcode 9.2 with `xcrun swift -version` returning Apple Swift version 4.0.3. Appears when trying to run it on an actual ios device. 

```
error: The following binaries use incompatible versions of Swift:
/Users/user/Library/Developer/Xcode/DerivedData/ReactNativeSampleApp-alidsejfzvavggbjvvdwzajefqfc/Build/Products/Debug-iphoneos/ReactNativeSampleApp.app/Frameworks/Onfido.framework/Onfido
/Users/user/Library/Developer/Xcode/DerivedData/ReactNativeSampleApp-alidsejfzvavggbjvvdwzajefqfc/Build/Products/Debug-iphoneos/ReactNativeSampleApp.app/Frameworks/SwiftyJSON.framework/SwiftyJSON
```

**Comment by anuragajwani** [comment]
@isnotahippy the current version (7.0.0) of the iOS SDK is not compatible with Xcode 9.2 and Swift 4.0.3, we recommend to upgrade Xcode version to 9.3.

**Comment by huangmr** [comment]
would be nice to see this to run with latest xcode...

**Comment by ngo275** [comment]
I made a similar PR #4 .

**Comment by rfreitas** [comment]
Closed since swift solution seems possible.

**Pull request closed without merging**